frontend/cotangles/src/pages/Calendar.tsx

```diff
@@ -0,0 +1,28 @@
+import "../App.css";
+import { CardTop, CardBody, CardHeader } from "@/components/ui/card";
+import { ContentWrapper } from "@/components/ui/content-wrapper";
+import { NavigationCotangles } from "@/components/ui/navigation-menu";
+// import { useParams } from "react-router-dom";
+function Calendar() {
+  // let { calendarId } = useParams();
+  // return <h1>Calendar ID {calendarId}</h1>;
+  return (
+    <>
+      <NavigationCotangles />
```

Comment:
potentially makes sense to move this into App.tsx instead of being at the top of each file

Reply:
actually we should do this - having it like this makes the button presses look odd

Reply:
OH IS THAT WHAT WAS CAUSING THE BUTTON PRESSES omg I was so confused for ages and then I just gave up on it and forgot about it :sob:

Reply:
oh wait actually we may be referring to a different thing for the buttons but I have moved it to app.tsx now 👍  actually didn't know u could do that, very cool